### PR TITLE
Implement waitlist component

### PR DESCRIPTION
**Pull request by aslife**
https://app.clickup.com/t/860qf2d3t

### CODE REVIEW

**Comment by aslife** [comment]
Solved conflicts

File: api/src/features/users/users.service.ts [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+    private readonly config: ConfigService
+  ) {}
+
+  async subscribe(email: string) {
+    console.log('Hello', this.config.get('MAILCHIMP_URL'), email)
+    try {
+      const request = await this.httpService.post(this.config.get('MAILCHIMP_URL'), { EMAIl: email }).toPromise()
+    } catch (e) {
+      console.log('Hello', e)
+    }
```
Comment (dmitriy-green) [inline api/src/features/users/users.service.ts:19]: Why do we need this? We can make a request from the frontend. No need to add this logic on the backend

File: frontend/app/features/assets/views/WaitList/WaitList.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    <div className={s.socialBlock}>
+      <div className={s.socialTitle}>
+        And join our community on social media to stay up to date.
+      </div>
+      <div className={s.socialBtnsBlock}>
+        <a href={'/'} className={s.socialLink}><TwitterIcon /></a>
+        <a href={'/'} className={s.socialLink}><DiscordIcon /></a>
+        <a href={'/'} className={s.socialLink}><GithubIcon /></a>
+        <a href={'/'} className={s.socialLink}><TelegramIcon style={{position: "relative", left: '-2px'}}/></a>
```
Comment (dmitriy-green) [inline frontend/app/features/assets/views/WaitList/WaitList.tsx:42]: `/`  path for the social links?

File: frontend/app/features/assets/views/WaitList/WaitList.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  date: Date
+}
+
+export const WaitList: FC<TProps> = ({onSubmit, date}) => {
+  const [emailField, setEmailField] = useState<string>('');
+  const [validEmail, setValidEmail] = useState<boolean>(false)
+
+  const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const email = e.target.value;
+    setValidEmail(validator.isEmail(email) ? true : false);
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/views/WaitList/WaitList.tsx:19]: Need to replace with existing validator.
Remove "validator" from package

File: api/package.json [hunk 4]
```diff
@@ -28,14 +28,15 @@
...
+    "@nestjs/axios": "^2.0.0",
     "@nestjs/common": "9.2.1",
     "@nestjs/config": "2.2.0",
     "@nestjs/core": "9.2.1",
     "@nestjs/event-emitter": "^1.4.1",
     "@nestjs/platform-express": "9.2.1",
     "@nestjs/typeorm": "9.0.1",
     "aws-sdk": "2.1281.0",
-    "axios": "^1.2.5",
+    "axios": "^1.3.5",
```
Comment (ruslanpalagin) [inline api/package.json:39]: why do you need axios & nestjs/axios ?
Can we remove it please?

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

**Review by ruslanpalagin — APPROVED** [review]
I am not happy with this pr.
Created task to process comments https://app.clickup.com/t/85zrwu758

**Pull request merged**